_std/defines/obj.dm

```diff
@@ -11,3 +11,5 @@
 /// At which alpha do opague objects become see-through?
 #define MATERIAL_ALPHA_OPACITY 190
 
+/// Overrides multicontext menu popup when an object is clicked by a player who is holding an item
+#define IGNORE_CONTEXT_CLICK_HELD 16
```

Comment:
why not `1<<3`? - also, this should be up with the object flags, no?

Reply:
i've used bitflags like once...i'm not sure of all the nuances of how they work, i'll change it.